### ISSUE TITLE
Illegal instruction

### ISSUE THREAD

**Issue by omaen** [mmaraya/port-mirroring#54]
When i try to use the port-mirroring application i get an error after a short time, "Illegal instruction", and then it stops.

```
`root@OpenWrt:~# port-mirroring --debug
2016-05-22 23:38:22[info] port-mirroring::main, mirroring_type:[remote][TEE], mirroring_source_num:[1], target:[192.168.3.100], filter:[], opt_promiscuous:[1].
2016-05-22 23:38:22[info] getSenderInterface, device=[eth1.3], mac=[c05627******].
2016-05-22 23:38:23[info] getRemoteARP, filter=[arp host 192.168.3.100], device=[eth1.3], remote mac=[326538******].
2016-05-22 23:38:23[info] port-mirroring::reopenSendHandle eth1.3 success.
2016-05-22 23:38:36[info] port-mirroring, 1000 packets mirrored.
Illegal instruction`
```

I currently use the package from [https://downloads.openwrt.org/snapshots/trunk/mvebu/generic/packages/](url)

My config:

```
config 'port-mirroring'
    option 'target' '192.168.3.100'
    option 'promiscuous' '1'
    option 'source_ports' 'br-lan'
    option 'filter' ''
    option 'protocol' 'TEE'
```

**Comment by omaen** [comment]
The result is the same for 'eth0', 'eth1' and 'wlan0'. I have IPv6 enabled in my system.
The router I'm using is Linksys WRT1900ACS.

**Comment by mmaraya** [comment]
Hi @ominator1! Thanks for reporting the issue. This is on my list of things to look into.

**Comment by mmaraya** [comment]
Sorry for the long delay in getting back to you on this issue.  https://downloads.openwrt.org/snapshots/trunk/mvebu/generic/packages/packages/ has version 1.4.2-1, which was released on 10/30/15. I have since found a couple of runtime errors (see 11dc009) which may fix the issue you're having. However, I haven't released that yet so it may be a few more weeks before it can be uploaded to OpenWrt. I'll keep this issue open until the next release.
